docs/bonsai-ui/checkout/index.mdx

```diff
@@ -147,9 +164,9 @@ type Config = {
 
   /* OPTIONAL */
 
-  /* A callback that will be invoked on checkout success */
+  /* Deprecated: Instead, please use onBonsaiEvent with "order_completed" tag. */
   onSuccess?: (order: Order) => void;
-  /* A callback that will be invoked when user submits shipping information */
+  /* Deprecated: Instead, please use onBonsaiEvent with "shipping_info_entered" tag. */
   onShippingSubmit?: (context: ShippingContext) => void;
```

Comment:
Going to keep these here but marked `deprecated` so that Editorialist doesn't feel like we're completely pretending this feature never existed.